test/quantization/test_quantize_fx.py

```diff
@@ -3962,3 +3992,8 @@ def test_resnet18_ddp(self):
         model = models.__dict__[name](pretrained=True).eval().float()
         self._test_model_impl(
             'ddp', 'resnet18', model, eager_quantizable_model)
+
+
+if __name__ == "__main__":
```

Comment:
we actually don't want to run this when people invoke the file could you change this to a runtime error before land: https://github.com/pytorch/pytorch/blob/master/test/quantization/test_quantize.py#L2330